Results bioRxiv Word-PCA (#11)
greenelab/annorxiver_manuscript: content/04.results.md

Tokens with an odds ratio greater than one are mainly centered on paper/figure references and research specific terms (Figure {@fig:biorxiv_pmc_pre_published_comp}).
Tokens with an odds ratio of less than one are focused on data availability, and research measurements such as number of cases and controls or significance testing (Figure {@fig:biorxiv_pmc_pre_published_comp}).

### Topic Analysis of bioRxiv's Principal Components

![
The top two principal components (PCs) consist of the following concepts: molecular vs quantitative biology (PC1) and genomics vs bioinformatics (PC2).
The left most plot (A) is a word cloud for the second PC.
Tokens in orange are associated with a PC's positive axis while tokens in blue are associated with the PC's negative axis.
The size of each token indicates the strength of association.
The word cloud at the bottom right (C) is for the first PC and follows the same trend as previously described.
The scatter plot at the top right (B) is a visualization of documents being plotted along the PC's axes.

> Documents were hand-picked based on the concepts captured by each PC.

Author: Categories were hand-picked. Changing text to reflect this point.

](https://raw.githubusercontent.com/greenelab/annorxiver/35d3ea0de3c9c78e3c524736bbaada00928c88fb/biorxiv/pca_association_experiment/output/pca_plots/figures/pca01_v_pca02_figure.png){#fig:pca_word_cloud_plot}

> The most predominant source of variation within bioRxiv preprints is the concept of molecular biology vs quantitative biology (Figure {@fig:pca_word_cloud_plot) C).

Reviewer: ```suggestion
We explored the document representations to understand the primary differences between the full text of bioRxiv preprints by performing principal components analysis of the document embeddings.
We visualized documents on principal components for selected components (Figure {@fig:pca_word_cloud_plot)B) and the loadings for each component (Figure {@fig:pca_word_cloud_plot)A,C).
The first principal component bioRxiv preprint document embeddings separated words frequent in molecular biology from those frequent in quantitative biology.
```

Reviewer: I tried to rearrange this so that figures are referred to in order. I couldn't _quite_ get it to work, but they are in the same sentence.

> This highlights the bisection of biomedical research where majority of results came from molecular biology experiments or quantitative biology experiments.

Reviewer: Do you want to add panels that show the average (+/- 95% CIs) for eigenvalues for preprints from each self-identified category on bioRxiv with respect to PCs 1 and 2?

The dotted line provides a breaking point between both categories.
The plot on the right (B) is a barchart of token frequency appearing in preprints and published versions of preprints respectively.
](https://raw.githubusercontent.com/danich1/annorxiver/f5d6c2d04e2fef0d38eaee77ef6c2a0b19ef6358/biorxiv/corpora_comparison/output/figures/preprint_published_comparison.png){#fig:biorxiv_pmc_pre_published_comp}

A preprint's linguistic style can change once a preprint has undergone the revision process prior to being published.
We quantified this linguistic difference by calculating the odds ratio of tokens appearing in the union in bioRxiv preprints and their published counterparts within PMC.
Tokens with an odds ratio greater than one are mainly centered on paper/figure references and research specific terms (Figure {@fig:biorxiv_pmc_pre_published_comp}).
Tokens with an odds ratio of less than one are focused on data availability, and research measurements such as number of cases and controls or significance testing (Figure {@fig:biorxiv_pmc_pre_published_comp}).

### Topic Analysis of bioRxiv's Principal Components

![

> The top two principal components (PCs) capture the concepts of molecular biology vs quantitative biology (PC1) and neuroscience vs bioinformatics (PC2).

Reviewer: ```suggestion
The top two principal components (PCs) appear to capture the concepts of molecular biology vs quantitative biology (PC1) and neuroscience vs bioinformatics (PC2).
```

Tokens in orange are most similar to a PC's positive direction while tokens in blue are most similar to a PC's negative direction.
The size of each token indicates the magnitude of the similarity score.
The scatter plot at the top right (B) is a visualization of documents being plotted along the PC directions.
Article categories were hand-picked based on the concepts captured by each PC.
](https://raw.githubusercontent.com/greenelab/annorxiver/35d3ea0de3c9c78e3c524736bbaada00928c88fb/biorxiv/pca_association_experiment/output/pca_plots/figures/pca01_v_pca02_figure.png){#fig:pca_word_cloud_plot}

![
Preprint categories have a diverse spread of quantitative and molecular biology results.
This is box plot shows preprints in each article category projected along the PC1 direction.
Negative values indicate molecular biology concepts, while positive values indicate quantitative biology concepts.
](https://raw.githubusercontent.com/danich1/annorxiver/4e0c90c590dd7958dc424a1b4fefcecccb3c28ad/biorxiv/pca_association_experiment/output/pca_plots/figures/category_box_plot_pc1.svg){#fig:pca1_pointplot}

![

> On average majority of bioRxiv preprint categories contain more neuroscience concepts than bioinformatic concepts.

Reviewer: How do we know this?

This highlights the bisection of biomedical research where majority of results can be categorized under the molecular biology category or the quantitative biology category.
Furthermore, this bisecting trend is evident across individual preprint categories as most categories lie on either side of the first principal component (Figure {@fig:pca1_pointplot}).
We also provide visual examples to reinforce this concept within Supplemental Tables {@tbl:top_five_pc1_table} and {@tbl:bottom_five_pc1_table}).
 
The second principal component represents the concept of neuroscience vs bioinformatics (Figure {@fig:pca_word_cloud_plot)A).
This principal component suggests that the bulk of preprints within bioRxiv are largely focused around neuroscience and bioinformatic concepts.
This split is evident in Figure {@fig:pca2_pointplot} as enriched categories along this principal component are quite related to neuroscience (negative end) or bioinformatics (positive end).
As with the first principal component we provide visual confirmation for the second principal component within Supplemental Tables {@tbl:top_five_pc2_table} and {@tbl:bottom_five_pc2_table}). 

> More principal component word clouds can be found within our online repository (see Data Availability).

Reviewer: Do you want to note that they are also available on the website?

](https://raw.githubusercontent.com/danich1/annorxiver/4e0c90c590dd7958dc424a1b4fefcecccb3c28ad/biorxiv/pca_association_experiment/output/pca_plots/figures/category_box_plot_pc1.svg){#fig:pca1_pointplot}

![
The second PC groups neuroscience related preprint categories and bioinformatics related preprint categories together.
This is box plot shows preprints in each article category projected along the PC2 direction.
Negative values indicate neuroscience concepts, while positive values indicate bioinformatic concepts.
](https://raw.githubusercontent.com/danich1/annorxiver/4e0c90c590dd7958dc424a1b4fefcecccb3c28ad/biorxiv/pca_association_experiment/output/pca_plots/figures/category_box_plot_pc2.svg){#fig:pca2_pointplot}

> | Title [citation]                                                                                                                                                  | PC_1              | License      | 

Reviewer: Wow - I'm amazed at how restrictive the licenses are. Can we link to figure 1 for each of them instead of embedding them?

| GpABC: a Julia package for approximate Bayesian computation with Gaussian process emulation [@doi:10.1101/769299]                                                 | 4.351517618262304 | CC-BY-NC-ND  | 
| SBpipe: a collection of pipelines for automating repetitive simulation and analysis tasks [@doi:10.1101/107250]                                                   | 4.321847854182741 |  CC-BY-NC-ND | 
| | | | | 
| Spatiotemporal proteomics uncovers cathepsin-dependent host cell death during bacterial infection [@doi:10.1101/455048]                                           | -4.263964235099807 |  CC-BY-ND   | 
| Systems analysis by mass cytometry identifies susceptibility of latent HIV-infected T cells to targeting of p38 and mTOR pathways [@doi:10.1101/371922]           | -4.279016673409032 | CC-BY-NC-ND | 
| NADPH consumption by L-cystine reduction creates a metabolic vulnerability upon glucose deprivation [@doi:10.1101/733162]                                         | -4.592209988884236 | None        | 

> | nhibition of Bruton’s tyrosine kinase reduces NF-kB and NLRP3 inflammasome activity preventing insulin resistance and microvascular disease [@doi:10.1101/745943] | -4.736613689905791 | None        | 

Reviewer: ```suggestion
| Inhibition of Bruton’s tyrosine kinase reduces NF-kB and NLRP3 inflammasome activity preventing insulin resistance and microvascular disease [@doi:10.1101/745943] | -4.736613689905791 | None        | 
```
